test/e2e/framework/test_context.go

```diff
@@ -399,7 +399,7 @@ func CreateGinkgoConfig() (types.SuiteConfig, types.ReporterConfig) {
 	// Randomize specs as well as suites
 	suiteConfig.RandomizeAllSpecs = true
 	// Disable skipped tests unless they are explicitly requested.
-	if len(suiteConfig.FocusStrings) == 0 && len(suiteConfig.SkipStrings) == 0 {
+	if len(suiteConfig.FocusStrings) == 0 && len(suiteConfig.SkipStrings) == 0 && suiteConfig.LabelFilter == "" {
```

Comment:
waiting for https://github.com/kubernetes/kubernetes/pull/125966 merged, just for testing, revert it when https://github.com/kubernetes/kubernetes/pull/125966 merged.